purchase_requisition_from_rfq/hooks.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api
+from odoo.addons.purchase_requisition.models.purchase_requisition import PurchaseOrder
+
+
+def post_load_hook():
+
+    @api.multi
+    def new_write(self, vals):
```

Comment:
I think you miss understand the point of using hook.
The hook mean, for the existing core odoo code that do not allow us to intercept, we would overwrite it here, to open the door for some intercept.

Reply:
Changed